### PR TITLE
AEPCore v3.2.1 release notes

### PR DESCRIPTION
**Pull request by kevinlind**
Release notes for AEPCore v3.2.1 and AEPIdentity v3.2.1

### CODE REVIEW

**Review by calebk1 — APPROVED** [review]
LGTM!

File: release-notes/README.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
...
 
+## June 21, 2021
+
+### iOS AEPCore 3.2.1
+
+* Update version for bundled ACPIdentity 3.2.1 release.
+
+## iOS AEPIdentity 3.2.1
+
+* Fixed a bug where `Identity.syncIdentifier` and `Identity.syncIdentifiers` APIs would always set authenticated state to `unknown`.
```
Comment (emdobrin) [inline release-notes/README.md:15]: Can we reword this to something like:
Fixed a bug where `Identity.syncIdentifier` and `Identity.syncIdentifiers` APIs would ignore the authentication state settings.

**Review by emdobrin — APPROVED** [review]
